libs/gui/src/dialog_base.cpp

```diff
@@ -17,23 +17,69 @@
  *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301,  USA.         *
  ***************************************************************************
  */
+#include <wx/display.h>
 #include <wx/dialog.h>
 #include <wx/sizer.h>
+#include <wx/string.h>
+#include <wx/html/htmlwin.h>
+#include <wx/stattext.h>
+#include <sstream>
```

Comment:
This breaks the rules for declarations https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

That is `<sstream>` should be on top.   While on it, you might want to keep the wx includes in lexical order.

Reply:
BTW: This header uses std::string and std:.stringstream. This means that the corresponding #include `<string>`  should be added - that in compiles now is just a coincidence. 

See: https://google.github.io/styleguide/cppguide.html#Self_contained_Headers